docs/UserGuide.md

```diff
@@ -182,7 +184,7 @@ Deletes the task at the given [index](#52-index) of the existing task list from
 
 Format: `delete INDEX`
 
-Example: `delete 3` deletes the 3rd task in Harmonia.
+Example: `delete 3` deletes the task at the third index of the task list.
```

Comment:
Should index here be linked to the glossary (with the explanation of what index is)? So "[index](#52-index)"